### PR TITLE
kernel: (streamer-alu) update streamer test to go end to end

### PR DESCRIPTION
**Pull request by jorendumoulin**
With all the new transforms, this implements an end-to-end test for an elementwise addition on the alu accelerator

### CODE REVIEW

File: kernels/streamer_alu/streamer_add_stream.preprocfinal.mlir
Comment (JosseVanDelm) [inline kernels/streamer_alu/streamer_add_stream.preprocfinal.mlir:1]: Hang on for a sec, this was working with dynamic shapes before?

**Review by JosseVanDelm — COMMENTED** [review]
Inclined to say approve, but I don't fully understand :)

**Comment by jorendumoulin** [comment]
Instead of starting from a handwritten .preprocfinal somewhere in the middle of the lowering flow, the new passes from other prs allow this one to be compiled from linalg all the way down!

File: kernels/streamer_alu/streamer_add_stream.preprocfinal.mlir
Comment (jorendumoulin) [reply kernels/streamer_alu/streamer_add_stream.preprocfinal.mlir:1]: Not really, a little bit down there are fixed sizes

Comment (JosseVanDelm) [reply kernels/streamer_alu/streamer_add_stream.preprocfinal.mlir:1]: Right! Okay!

**Review by JosseVanDelm — APPROVED** [review]
Cool dude!

**Pull request merged**